src/app/components/profile/profile.component.html

```diff
@@ -85,11 +75,21 @@
 								{{user.google_scholar_url}}
 								</div>
 							    </div>
+                </li>
```

Comment:
I am not able to understand this change, could you please explain.

Reply:
The order should be Google Scholar, GitHub, and LinkedIn. The order was not correct in the HTML and so I moved the positions.